File: frameworks/Python/webware/app/Context/plaintext.py

from WebKit.HTTPContent import HTTPContent

class plaintext(HTTPContent):
    def defaultAction(self):
    	self.response().clearHeaders()
        self.response()._headers["Content-Type"] = "text/plain"
        output = "Hello, World!"
        self.response()._headers["Content-Length"] = len(output)
        self.write(output)

File: frameworks/C++/webframework_cxx/benchmark/src/PlainText.cpp
#include "PlainText.h"

namespace executor
{
	void PlainText::doGet(framework::HttpRequest& request, framework::HttpResponse& response)
	{
		response.addHeader("Content-Type", "text/plain");

		response.setBody("Hello, World!");
	}

	DEFINE_EXECUTOR(PlainText);
}
